Extensions/TextInput/JsExtension.js

```diff
@@ -216,6 +249,9 @@ module.exports = {
       borderWidth: 1,
       readOnly: false,
       disabled: false,
+      padding: 0,
+      textAlign: 'left',
+      maxLength: '20',
```

Comment:
Should it be a string here? Also, should you use -1/0 instead as an initial value?